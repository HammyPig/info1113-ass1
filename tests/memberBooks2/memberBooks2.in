member books 100000
ADD MEMBER you, the marker (yes you)
add member YES YOU!
member books 100000
add collection books.csv
rent 100000 111111
rent 100001 111112
rent 100000 111112
member books 100000
relinquish 100000 111111
member books 100000
rent 100000 111112
rent 100000 111113
rent 100000 111114
rent 100000 111115
rent 100000 111116
member books 100000
relinquish all 100000
member books 100000