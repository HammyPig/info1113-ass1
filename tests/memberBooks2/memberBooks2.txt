recognises if no members exist, books appear when successfully rented and are removed when successfully relinquished