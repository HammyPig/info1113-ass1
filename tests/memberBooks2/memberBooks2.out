user: No members in system.

user: Success.

user: Success.

user: Member not currently renting.

user: 17 books successfully added.

user: Success.

user: Success.

user: Book is currently unavailable.

user: To Kill a Mockingbird (Harper Lee)

user: Success.

user: Member not currently renting.

user: Book is currently unavailable.

user: Success.

user: Success.

user: Success.

user: Success.

user: The Iliad (Homer)
The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: Success.

user: Member not currently renting.

user: Ending Library process.
